charts/spire/charts/spire-server/values.yaml

```
@@ -167,7 +167,9 @@ dataStore:
     username: spire
     ## @param dataStore.sql.password Only used when type != "sqlite3"
     password: ""
-    ## @param dataStore.sql.options [array] Only used when type != "sqlite3"
+    ## @param dataStore.sql.file Only used when type == "sqlite3"
+    file: ""
+    ## @param dataStore.sql.options [array]
```

```suggestion
    ## @param dataStore.sql.options [array] takes an array of objects of form {<key>: <value>} to use when building the database connection string
```

idea for improvement of options documentation (obviously need to regen the readme if accepted)